psu/psu.c

```diff
@@ -159,6 +160,12 @@ static int sdp_writeFile(hid_device *dev, uint32_t addr, void *data, size_t size
 		/* print progress */
 		fprintf(stderr, "\r - Sent (%lu/%lu) %3.0f%% ", offset, size, ((float)offset / (float)size) * 100.0f);
 
+		/* Report 2 size has to be align to 16, information define in HID Report Descriptor - ID 2 */
```

Comment:
Really appreciate the comment regarding **why** this code is needed :). 
Btw, align -> aligned